test/integration/tests/import.sh

```diff
@@ -70,7 +73,29 @@ openssl enc -in plain.enc -out plain.dec.ssl -d -K `xxd -p sym.key` -iv 0 \
 
 diff plain.txt plain.dec.ssl
 if [ $? != 0 ];then
-echo "TEST: tpm2_import failed"
+echo "TEST: tpm2_import failed for Symmetric Key"
+exit 1
+fi
+
+#Asymmetric Key Import Test
+openssl genrsa -out private.pem 2048
+openssl asn1parse -in private.pem -strparse 533 -out priv.bin -noout
+openssl asn1parse -in private.pem -strparse 7 -out pub.bin -noout
+
+tpm2_import -Q -G rsa -K pub.bin -k priv.bin -H 0x81010005 -f parent.pub \
+-q import_rsa_key.pub -r import_rsa_key.priv
+
+tpm2_load -Q -H 0x81010005 -u import_rsa_key.pub -r import_rsa_key.priv \
+-n import_rsa_key.name -C import_rsa_key.ctx
+
+openssl rsa -in private.pem -out public.pem -outform PEM -pubout
+openssl rsautl -encrypt -inkey public.pem -pubin -in plain.txt -out plain.rsa.enc
+
+tpm2_rsadecrypt -c import_rsa_key.ctx -I plain.rsa.enc -o plain.rsa.dec
+
+diff plain.txt plain.rsa.dec
+if [ $? != 0 ];then
+echo "TEST: tpm2_import failed for Asymmetric Key"
 exit 1
```

Comment:
You shouldn't need this if block. The tests have the trap ERR set, so on error that trap handler gets called and prints the line number of the failed command and exits 1.

Reply:
@williamcroberts IIUC I need to replace the commented block with "trap onerror ERR" or is it that, it is set by default so just remove line 97-99

Reply:
It's already set, so you just need diff. Ill fix this up for you today and get in the repo. You did enough here.

Reply:
hehe.. thanks 👍